### DISCUSSION TITLE
关于自己部署音乐API

### DISCUSSION THREAD

**Discussion by Kirara-Tianqing** [imsyy/home#173] [General]
大佬您好，我这边在音乐API的使用上遇到了问题想请教一下：我按照您给出的https://github.com/xizeyoupan/Meting-API#deno-deploy 项目，自己在本地服务器(网页和API都在同一台服务器上)上用docker成功部署了API接口，容器映射的端口是3000，请问我在.env文件中该如何填写音乐API地址？

**Comment by imsyy** [comment]
把端口号加在后面就行

**Reply by Kirara-Tianqing** [reply]
我填写的是 http://localhost:3000/api ，但是我构建完成后网页ctrl+F5刷新无法加载出音乐播放器，而且我不知道是不是因为无法加载出音乐播放器的原因，有时候服务器直接无法连接，只能重启服务器后才能连接

**Reply by Kirara-Tianqing** [reply]
最后我做了反代https://域名/api 才能使用我自建的API， 为什么填写这个http://localhost:3000/api 地址无法加载播放器还是一头雾水

**Reply by imsyy** [reply]
生产环境肯定无法访问 localhost 呀 😂

**Comment by Kirara-Tianqing** [comment]
![image](https://github.com/imsyy/home/assets/81161427/9737e06f-93be-4653-b0bc-6021b32c182a)
